CMakeLists.txt

```diff
@@ -875,6 +875,10 @@ if (HDF5_ENABLE_THREADSAFE)
   if (Threads_FOUND)
     set (H5_HAVE_THREADSAFE 1)
   endif ()
+else()
+  # Always check for pthreads for VFD SWMR (for now)
+  set (THREADS_PREFER_PTHREAD_FLAG ON)
+  find_package (Threads REQUIRED)
```

Comment:
VFD SWMR requires background threads?   (That's a potential issue for some customers)

Reply:
No. It's necessary on MacOS to run some of the tests. They were written to use sigtimedwait, which is not present on MacOS.